macos-init.sh

```diff
@@ -23,11 +23,6 @@ set -e
 
 # base conmmon cli tools
 base_cli=(
```

Comment:
what if we just remove `base_cli` since it looks like it wont be needed?

Reply:
good call